app/helpers/phases_helper.rb

```diff
@@ -9,4 +9,26 @@ def phase_number(challenge, phase)
   def phase_has_recused_evaluator?(phase)
     phase.evaluator_submission_assignments.recused.exists?
   end
+
+  def statuses
+    {
+      draft: "Draft",
+      gsa_review: "GSA review",
+      approved: "Approved",
+      edits_requested: "Edits requested",
+      unpublished: "Unpublished",
+      published: "Published",
+      archived: "Archived"
+    }
+  end
+
+  def evaluation_status(phase)
+    if phase.submissions.eligible_for_evaluation.where.not(status: [:in_progress, :completed]).exists?
```

Comment:
"Not Started" should include phases with no eligible submissions yet (so they don't show up as in progress)
```suggestion
    if !phase.submissions.eligible_for_evaluation.exists? || phase.submissions.eligible_for_evaluation.where.not(status: [:in_progress, :completed]).exists?
```

Reply:
I also realized the second part of this query isn't correct:
```
phase.submissions.eligible_for_evaluation.where.not(status: [:in_progress, :completed]).exists?
```
this will find eligible submissions where the status is not either `in_progress` or `completed`, in other words it will find submissions that are `not_started` status. what you really want is to negate the whole query
```suggestion
    if !phase.submissions.eligible_for_evaluation.where(status: [:in_progress, :completed]).exists?
```
so that says eligible submissions where the status == `in_progress` or status == `completed` do not exist. (also the field is `evaluation_status`, so summing it all up this line should be
```suggestion
    if !phase.submissions.eligible_for_evaluation.exists? || !phase.submissions.eligible_for_evaluation.where(evaluation_status: [:in_progress, :completed]).exists?
```
this method needs some specs to validate it's working. it's too complicated and important to rely on visual review, which is easy for me to make mistakes.